src/main/resources/application.properties

```
@@ -12,6 +12,9 @@ scanner=scanoss
 # Corresponding env. variable LPVS_LICENSE_CONFLICT
 license_conflict=db
 
+# URL of the external OSORI license database. Default is empty.
+license_source=http://223.255.204.223:8082
```

I think we can use `https://olis.or.kr:8082/` instead of static ip and http protocol. What do you think?
```suggestion
license_source=https://olis.or.kr:8082
```